main/src/ca/uwaterloo/flix/language/phase/Parser.scala

```diff
@@ -600,8 +612,8 @@ class Parser(val source: Source) extends org.parboiled2.Parser {
     }
 
     def Primary: Rule1[ParsedAst.Expression] = rule {
-      LetRec | LetMatch | IfThenElse | Match | LambdaMatch | Switch | Unsafe | Native | Lambda | Tuple |
-        FNil | FSet | FMap | Literal |
+      LetRec | LetMatch | IfThenElse | Match | LambdaMatch | Switch | Unsafe | Native | Lambda | Tuple | ArrayLit |
+        ArrayNew | ArrayLength |VecLength | VecLit | VecNew | FNil | FSet | FMap | Literal |
```

Comment:
Any reason that they occur in the order: ArrayLit, ArrayNew, ArrayLength, but then in the order VecLength, VecLit, VecNew?